docs/installation.md

```diff
@@ -17,13 +17,6 @@ cd ngi_reports
 python setup.py install
```

Comment:
```suggestion
pip install -e .
```
I'm changing to recommend installation by `pip` for everything now, it's more up to date and causes fewer weird edge cases with package version resolution (eg. `python setup.py` sometimes picks random pre-releases).

Reply:
Done!